krakus/utils/wios.py

```diff
@@ -0,0 +1,37 @@
+class StationDataInterface:
+    """
+    Main interface for station data gathered by the WIOS API.
+    Every instance will represent a single station, so if you have more
+    than one station in the API response, you should instantiate this class
+    multiple times.
+    """
+
+    def __init__(self, station_data):
+        self._station_data = station_data
+
+    @property
+    def data(self) -> dict:
+        return self._station_data['data']
+
+    @property
+    def title(self) -> str:
+        return self.data['title']
+
+    @property
+    def series(self) -> list:
+        return self.data['series']
+
+    def _filter_series(self, key: str) -> list:
+        try:
+            raw = next(filter(lambda x: x['paramId'] == key, self.series))
+            return raw['data']
+        except StopIteration:
+            return []
+
+    @property
+    def pm10_data(self) -> list:
+        return self._filter_series('pm10')
+
+    @property
+    def pm25_data(self) -> list:
+        return self._filter_series('pm2.5')
```

Comment:
Two things came to my mind:
1. How about exposing date as a property? We could either use startTime/extStartTime from Wios API response or do some regex on the title (I'm assuming title is always there). 
2. Another thing that could be useful for analysis would be having station name as property. I don't see it in API response though - so could be extracted from title as well?

Reply:
1. Yep, having it as datetime.date or even string alone would enable convenient filtering of dataset.

Reply:
sure, np, will add it

Reply:
@jszafran 1) and 2) have been targeted, for the repr --> https://github.com/MattBlack85/krakus/issues/21

please re-review